scripts/forklift.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Sometimes we need to forklift production, so this attempts to do it nicely.
+set -x
+HOST_PATTERN="iemwebfarm,iem_appliance,anticyclone.agron.iastate.edu"
+INVENTORY_DIR="/home/akrherz/projects/infra-ansible/inventories"
+
+for machine in $(ansible $HOST_PATTERN -i $INVENTORY_DIR --list-hosts | awk 'NR>1 {print $1}'); do
+  # Enable the F5 denial script
+  ssh root@$machine "sh /opt/iemwebfarm/scripts/f5util.sh ON"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)